packages/playwright/src/prompt.ts

```diff
@@ -119,10 +119,13 @@ export async function attachErrorPrompts(testInfo: TestInfo, sourceCache: Map<st
       );
     }
 
+    const promptPath = testInfo.outputPath(`_prompt-${index}.md`);
```

Comment:
The file name does not have to start with the underscore 😄

Reply:
good point! and even better, it doesn't need to have an index if there's only one. fixed in https://github.com/microsoft/playwright/pull/35341/commits/d1cc9e9c3c755420545dd327601ad5b2fef13ac4.